app/lib/smooch_nlu.rb

```diff
@@ -44,15 +44,11 @@ def update_keywords(language, keywords, keyword, operation, doc_id, context)
     }
     if operation == 'add' && !keywords.include?(keyword)
       keywords << keyword
-      alegre_operation = 'post'
-      alegre_params = common_alegre_params.merge({ text: keyword, models: ALEGRE_MODELS_AND_THRESHOLDS.keys })
+      Bot::Alegre.get_sync_with_params(common_alegre_params.merge({ text: keyword, models: ALEGRE_MODELS_AND_THRESHOLDS.keys }), "text")
```

Comment:
Here we should be _indexing_ content... but looks like we're retrieving?

Reply:
I've just added aliases and renamed per our conversation this morning - let me know how it looks to your eyes!